nengo/spa/tests/test_basalganglia.py

```diff
@@ -22,6 +22,7 @@ def test_basal_ganglia(Simulator, seed, plt):
             'dot(vision,CAT)+0.5-dot(vision,CAT) --> motor=E',
             'dot(vision,PARROT) + compare --> motor=F',
             '0.5*dot(vision,MOUSE) + 0.5*compare --> motor=G',
+            '( dot(vision,MOUSE) - compare) * 0.5 --> motor=H'
```

Comment:
Nope, I just thought it made it more legible
On 27 Aug 2015 17:35, "Jan Gosmann" <notifications@github.com> wrote:

> In nengo/spa/tests/test_basalganglia.py
> <https://github.com/nengo/nengo/pull/823#discussion_r38149082>:
>
> > @@ -22,6 +22,7 @@ def test_basal_ganglia(Simulator, seed, plt):
> >              'dot(vision,CAT)+0.5-dot(vision,CAT) --> motor=E',
> >              'dot(vision,PARROT) + compare --> motor=F',
> >              '0.5*dot(vision,MOUSE) + 0.5*compare --> motor=G',
> > +            '( dot(vision,MOUSE) - compare) * 0.5 --> motor=H'
>
> Does the space after the opening parenthesis have any significance?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nengo/nengo/pull/823/files#r38149082>.
>